app/frontend/src/components/FileInput.vue

```diff
@@ -348,7 +348,8 @@ export default {
     },
     updateContexts(obj) {
       try {
-        this.form.contexts = JSON.stringify(obj);
+        // create a JSON object of context(s)
+        this.form.contexts = Array.isArray(obj) ? JSON.stringify(obj[0]) : JSON.stringify(obj);
```

Comment:
We've verified that contexts will not accept an array. We should have our validation rules updated to reject arrays, and roll this back to the original JSON.stringify only assignment. A validation rule along the lines of
```js
v => !Array.isArray(v) || 'Should not be an array'
```
would suffice.